pkg/server/server.go

```diff
@@ -15,6 +16,11 @@ import (
 var log = logger.Package()
 
 func Run(ctx context.Context, c services.Config) error {
+	go func() {
```

Comment:
Do we want this to be always on? now that we have a released version, we probably want this off by default so that people aren't accidentally exposing debug info to the world.

Reply:
I'm a fan of leaving it on in production, because that's where we're most likely to run into issues that will require profiling info to resolve.

Is there's some other way to ensure we don't accidentally expose the port publicly.

Reply:
it is fine, I can only run it behind an env variable 👍